budget/_vars-budget.html

```diff
@@ -3,7 +3,7 @@
 {% set nav_items = [
     (path, 'index', 'Budget and Strategy'),
     (path + 'strategic-plan/', 'strategic-plan', 'Strategic Plan'),
-    (path + 'performance-plan-report/', 'performance-plan-report', 'Performance Plan Report'),
-    (path + 'financial-report/', 'financial-report', 'Financial Report & Update'),
-    (path + 'funding-request/', 'funding-request', 'Funding Request'),
+    (path + 'performance-plan-report/', 'performance-plan-report', 'Performance Plan & Report'),
```

Comment:
This one should stay singular. Performance Plan and Report is actually a single document, released annually. Pluralizing makes it seem like separate documents. The Financial Reports and Updates, however, are separate documents, and pluralizing them, for whatever reason, implies they are separate. I would be more concerned with the incongruity if anything under PP&R came out more often than once a year.